crates/bdk/src/wallet/mod.rs

```diff
@@ -1431,7 +1430,6 @@ impl<D> Wallet<D> {
         };
 
         let (fee_rate, mut fee_amount) = match params.fee_policy.unwrap_or_default() {
-            //FIXME: see https://github.com/bitcoindevkit/bdk/issues/256
```

Comment:
📌 Removed because issue said it is fixed by 1.0 work but is it? I think we're supposed to make sure both the absolute fee amount AND fee rate is at least as big or bigger than the original transaction but I think we're only checking one or the other based on the `FeePolicy`.

Reply:
I'm re-opening #256 since it seems it isn't fixed.